python/Completion.py

```diff
@@ -8,18 +8,31 @@ def get_completions(self, column, partialWord):
         parameters['WantDocumentationForEveryCompletionResult'] = \
             bool(int(vim.eval('g:omnicomplete_fetch_full_documentation')))
 
+        want_snippet = \
+            bool(int(vim.eval('g:omnicomplete_want_snippet')))
+
+        want_method_header = \
+            bool(int(vim.eval('g:omnicomplete_want_method_header')))
+
+        want_return_type = \
+            bool(int(vim.eval('g:omnicomplete_want_return_type')))
```

Comment:
Please use `g:OmniSharp_` prefix for config variable.
`g:omnicomplete_fetch_full_documentation` exists already, but this is wrong. I'll fix it.